backend/app/tests/services/llm/test_jobs.py

```
@@ -1156,6 +1164,207 @@ def test_execute_job_continues_when_no_validator_configs_resolved(
+            mock_session.return_value.__exit__.return_value = None
+
+            mock_chain_record = MagicMock()
+            mock_chain_record.id = chain_id
+            mock_create_chain.return_value = mock_chain_record
+            mock_creds.return_value = None
+            mock_handle_error.return_value = {
+                "success": False,
+                "error": "Unexpected error occurred",
+            }
+
+            result = self._execute_chain_job(chain_request_data)
```

Ruff (0.15.2)

[error] Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)